.github/workflows/ci.yaml

```diff
@@ -12,9 +12,14 @@ on:
       branches:
          - "main"
          - "release/*"
+      tags:
+         - "v**"
 
    workflow_dispatch:
 
+   release:
+      types: [ published ]
```

Comment:
What does this part do?

Reply:
The idea is to trigger this workflow when there's a new release https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

I'm not sure how this interacts with the `tags: "v**"` trigger if the release also creates a tag... I hope that `ci.yml` only gets triggered once.

Reply:
Gotcha :)